test/e2e/conftest.py

```diff
@@ -105,3 +105,26 @@ def check_created_and_modified_times_updated_correctly(post_response: Response,
         assert datetime.fromisoformat(updated_data["modified_time"]) > datetime.fromisoformat(
             original_data["modified_time"]
         )
+
+    @staticmethod
+    def replace_unit_values_with_ids_in_properties(data: dict, unit_value_id_dict: dict[str, str]) -> dict:
```

Comment:
I think effectively yes, right now the one inside the class is used in all new tests, the old one outside is slightly different and is just used in the old tests. So I didn't just replace as it would require updating all of the catalogue items and items tests which can change when they are refactored instead.

Reply:
In that case we can remove it at the end and sorry I was meant to ask if we can remove it rather than move it 🤦‍♂️

Reply:
No worries